### DISCUSSION TITLE
- DACSS601 Final Project

### DISCUSSION THREAD

**Discussion by giscus** [DACSS/DACSS_601_August2022_v2#261] [Announcements]
# - DACSS601 Final Project

Analysis of Olympics data from 1896-present

https://dacss.github.io/DACSS_601_August2022_v2/posts/FinalProject_ManiShankerKamarapu.html

**Comment by LarriMill** [comment]
Hi Mani Shanker, really nice work! You do a great job introducing your reader to the topic, your code is easy to read, and your visuals are great. My one piece of advice is that it's good to check out your data when the patterns you are seeing are significantly disrupted, like you saw around 2000. It turns out, that this is when the count was likely divided into summer and winter cycles, rather than an aggregate of both like in previous years.
